tests/GltfTests.cpp

```diff
@@ -0,0 +1,150 @@
+            CHECK(gltfUtil::getAlphaMode(mat) == expectedResults["alphaMode"].as<int>());
+            CHECK(gltfUtil::getAlphaCutoff(mat) == expectedResults["alphaCutoff"].as<float>());
+            CHECK(gltfUtil::getBaseAlpha(mat) == expectedResults["baseAlpha"].as<float>());
+            CHECK(gltfUtil::getMetallicFactor(mat) == expectedResults["metallicFactor"].as<float>());
+            CHECK(gltfUtil::getRoughnessFactor(mat) == expectedResults["roughnessFactor"].as<float>());
+            CHECK(gltfUtil::getBaseColorTextureWrapS(model, mat) == expectedResults["baseColorTextureWrapS"].as<int>());
+            CHECK(gltfUtil::getBaseColorTextureWrapT(model, mat) == expectedResults["baseColorTextureWrapT"].as<int>());
+
+            CHECK(gltfUtil::getBaseColorFactor(mat) == expectedResults["baseColorFactor"].as<std::vector<float>>());
+            CHECK(gltfUtil::getEmissiveFactor(mat) == expectedResults["emissiveFactor"].as<std::vector<float>>());
+        }
+
+        // Accessor smoke tests
+        cesium::omniverse::PositionsAccessor positions;
+        cesium::omniverse::IndicesAccessor indices;
+        CHECK_NOTHROW(positions = gltfUtil::getPositions(model, prim));
+        CHECK_NOTHROW(indices = gltfUtil::getIndices(model, prim, positions));
+        if (gltfUtil::hasNormals(model, prim, true)) {
+            CHECK_NOTHROW(gltfUtil::getNormals(model, prim, positions, indices, false));
+        }
+        if (gltfUtil::hasVertexColors(model, prim, 0)) {
+            CHECK_NOTHROW(gltfUtil::getVertexColors(model, prim, 0));
+        }
+        if (gltfUtil::hasTexcoords(model, prim, 0)) {
+            CHECK_NOTHROW(gltfUtil::getTexcoords(model, prim, 0));
+        }
+        if (gltfUtil::hasImageryTexcoords(model, prim, 0)) {
+            CHECK_NOTHROW(gltfUtil::getImageryTexcoords(model, prim, 0));
+        }
+        CHECK_NOTHROW(gltfUtil::getExtent(model, prim));
+
+        // Default getter smoke tests
+        CHECK_NOTHROW(gltfUtil::getDefaultBaseAlpha());
+        CHECK_NOTHROW(gltfUtil::getDefaultBaseColorFactor());
+        CHECK_NOTHROW(gltfUtil::getDefaultMetallicFactor());
+        CHECK_NOTHROW(gltfUtil::getDefaultRoughnessFactor());
+        CHECK_NOTHROW(gltfUtil::getDefaultEmissiveFactor());
+        CHECK_NOTHROW(gltfUtil::getDefaultWrapS());
+        CHECK_NOTHROW(gltfUtil::getDefaultWrapT());
+        CHECK_NOTHROW(gltfUtil::getDefaultAlphaCutoff());
+        CHECK_NOTHROW(gltfUtil::getDefaultAlphaMode());
+    }
+
+    TEST_CASE("check helper functions on various models") {
+
+        std::vector<std::string> gltfFiles;
+
+        // get list of gltf test files
+        for (auto const& i : std::filesystem::directory_iterator(ASSET_DIR)) {
+            std::filesystem::path fname = i.path().filename();
+            if (fname.extension() == ".gltf" || fname.extension() == ".glb") {
+                gltfFiles.push_back(fname.string());
+            }
+        }
```

Comment:
I think `gltfFiles` could be a vector of `std::filesystem::path` instead

Reply:
it eventually leads to a string based key/value lookup in the broader testUtils, which uses std::string to be more general than assuming all scenario names will be file paths. I can certainly stringify the path later in this function, but I think it needs to become a string before `getScenarioConfig` reads it.